libr/bin/p/bin_elf.inc

```diff
@@ -569,12 +569,15 @@ static RList* libs(RBinFile *bf) {
 
 static RBinReloc *reloc_convert(struct Elf_(r_bin_elf_obj_t) *bin, RBinElfReloc *rel, ut64 GOT) {
 	r_return_val_if_fail (bin && rel, NULL);
+	if (GOT == UT64_MAX) {
```

Comment:
1) why UT64_MAX should be here? If it is, probably the code that puts it there should be fixed
2) why if got == ut64_max you transform it into got = 0?

Reply:
ignore this pr the ut64 change is in another pr, ill rebase and cleanup when the others get merged